detection_rules/etc/version.lock.json

```diff
@@ -8127,11 +8523,20 @@
     "version": 104
   },
   "e26f042e-c590-4e82-8e05-41e81bd822ad": {
-    "min_stack_version": "8.3",
+    "min_stack_version": "8.12",
+    "previous": {
+      "8.3": {
+        "max_allowable_version": 211,
+        "rule_name": "Suspicious .NET Reflection via PowerShell",
+        "sha256": "a85be96f9a8185ce72aee9271706a90a0667bc9dc8340ec37a74fc874c3ba6d9",
+        "type": "query",
+        "version": 112
+      }
+    },
     "rule_name": "Suspicious .NET Reflection via PowerShell",
-    "sha256": "a85be96f9a8185ce72aee9271706a90a0667bc9dc8340ec37a74fc874c3ba6d9",
+    "sha256": "3cf8ff583ef123ebe0ef752da349e94652bcd203d089689bf6cfba36e727cc9d",
     "type": "query",
-    "version": 112
+    "version": 212
```

Comment:
Similar to the other bump of the hundreds place, but this bump does not change any of the other numbers? May be nothing, just noting here in case it is relevant.

Reply:
@eric-forte-elastic I think these are due to the minstack version changes that would have the hundreds place bump.  Rule Suspicious .NET Reflection via PowerShell had a minstack version change to 8.12.0 in this release.

Reply:
That part makes sense, but my concern is that:
`Rule Suspicious .NET Reflection via PowerShell` went from 112 -> 212
but 
`"Microsoft 365 User Restricted from Sending Email",` went from 105 -> 206

Reply:
This is expected. The reason they slightly differ is because it is a build time field update that we are reacting to, vs a rule author introducing a fork.